packages/calcite-components/src/components/scrim/scrim.tsx

```diff
@@ -116,4 +120,35 @@ export class Scrim implements LocalizedComponent, T9nComponent {
       </div>
     );
   }
+
+  // --------------------------------------------------------------------------
+  //
+  //  Private Methods
+  //
+  // --------------------------------------------------------------------------
+
+  private storeLoaderEl = (el: HTMLCalciteLoaderElement): void => {
+    this.loaderEl = el;
+    this.handleResize();
+  };
+
+  private getScale(size: number): Scale {
+    if (size < BREAKPOINTS.s) {
+      return "s";
+    } else if (size >= BREAKPOINTS.l) {
+      return "l";
+    } else {
+      return "m";
+    }
+  }
+
+  private handleResize = (): void => {
```

Comment:
This can be a class method (instead of an assigned bound function) because its call sites already bind `this` to the instance.